### PR TITLE
fix: remove non-transporter genes

### PR DESCRIPTION
**Pull request by haowang-bioinfo**
### Main improvements in this PR:
* remove **22** non-transporter genes from `Transporter reactions`, as proposed in #444

**I hereby confirm that I have:**

- [X] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch

### CODE REVIEW

**Comment by haowang-bioinfo** [comment]
Note that removal of CLTRN (ENSG00000147003) and SLC6A18 (ENSG00000164363) in this PR led to 14 amino acid transport reactions with empty GPR rules.

These 14 reactions are retained as such. Further changes will be conducted when actual transporters are identified.

File: model/Human-GEM.yml [hunk 1]
```diff
@@ -172044,7 +172044,7 @@
           - MAM02751c: 1
       - lower_bound: 0
       - upper_bound: 1000
-      - gene_reaction_rule: "(ENSG00000101892 and ENSG00000137731 and ENSG00000163399) or (ENSG00000018625 and ENSG00000129244 and ENSG00000137731) or (ENSG00000018625 and ENSG00000069849 and ENSG00000137731) or (ENSG00000101892 and ENSG00000105409 and ENSG00000137731) or (ENSG00000105409 and ENSG00000137731 and ENSG00000143153) or (ENSG00000105409 and ENSG00000129244 and ENSG00000137731) or (ENSG00000069849 and ENSG00000105409 and ENSG00000137731) or (ENSG00000101892 and ENSG00000132681 and ENSG00000137731) or (ENSG00000132681 and ENSG00000137731 and ENSG00000143153) or (ENSG00000129244 and ENSG00000132681 and ENSG00000137731) or (ENSG00000069849 and ENSG00000132681 and ENSG00000137731) or (ENSG00000137731 and ENSG00000143153 and ENSG00000163399) or (ENSG00000129244 and ENSG00000137731 and ENSG00000163399) or (ENSG00000069849 and ENSG00000137731 and ENSG00000163399) or (ENSG00000018625 and ENSG00000101892 and ENSG00000137731) or (ENSG00000018625 and ENSG00000137731 and ENSG00000143153)"
```
[the remark this answers is not in the document]
Comment (johan-gson) [reply model/Human-GEM.yml:172047]: In theory, it could be that this one is the transporter and the others are signaling enzymes that activate it.

Comment (johan-gson) [inline model/Human-GEM.yml:172047]: The first and you have removed here contains ENSG00000137731, which is not in the remove list in issue 444. I'm not sure if the other two are signaling enzymes that activates this enzyme, or how this works, but please check that this is correct.

Comment (johan-gson) [reply model/Human-GEM.yml:172047]: The same goes for ENSG00000163399. Looking this over is probably good.

File: model/Human-GEM.yml [hunk 2]
```diff
@@ -172506,7 +172506,7 @@
...
```
Comment (johan-gson) [inline model/Human-GEM.yml:172509]: Same thing here

File: model/Human-GEM.yml [hunk 3]
```diff
@@ -249224,8 +249224,8 @@
           - MAM02519c: 1
           - MAM02519e: -1
       - lower_bound: 0
-      - upper_bound: 1000
-      - gene_reaction_rule: "ENSG00000147003 and ENSG00000164363"
```
Comment (johan-gson) [inline model/Human-GEM.yml:249228]: ENSG00000147003 is not listed in issue 444.

File: model/Human-GEM.yml [hunk 4]
```diff
@@ -249264,8 +249264,8 @@
...
```
Comment (johan-gson) [inline model/Human-GEM.yml:249268]: Same here

File: model/Human-GEM.yml [hunk 5]
```diff
@@ -249284,8 +249284,8 @@
...
```
Comment (johan-gson) [inline model/Human-GEM.yml:249288]: and here

File: model/Human-GEM.yml [hunk 6]
```diff
@@ -249362,8 +249362,8 @@
           - MAM02519c: 1
           - MAM02519e: -1
       - lower_bound: 0
-      - upper_bound: 1000
```
Comment (johan-gson) [inline model/Human-GEM.yml:249365]: and here

Comment (johan-gson) [reply model/Human-GEM.yml:249365]: And a whole bunch of places further down

**Review by johan-gson — COMMENTED** [review]
Can you take a look at the comments I added - I some of the removed genes are not in the removal list in issue 444.

**Comment by haowang-bioinfo** [comment]
good job @johan-gson, you correctly pinpointed the mistakes I made, now close this to fix the problme.

**Comment by haowang-bioinfo** [comment]
The removal of _ENSG00000137731_ and _ENSG00000163399_ is due to ENSG00000101892, while _ENSG00000147003_ was removed because of ENSG00000164363. They are subunits of two complexes that are striped away as a whole when any subunit is deleted.

To avoid this unexpected outcome, ENSG00000101892 and   ENSG00000164363 are taken away from the list in #444, while  the updated change is implemented afresh in #450.

**Pull request closed without merging**